src/styles/blocks/card.scss

```diff
@@ -0,0 +1,89 @@
+.card {
+  width: $card-width;
+  padding: 32px 16px 16px;
+  background-color: $white-color;
+  border: 1px solid $grey-color;
+  border-radius: 5px;
+  margin-right: $card-margin-right;
```

Comment:
BEM block can't have margins

Reply:
Yes, you are right, I will put it in the catalog. +